package.json

```diff
@@ -32,8 +32,9 @@
   "devDependencies": {
     "@biomejs/biome": "^1.9.4",
     "@zowe/secrets-for-zowe-sdk": "^8.10.4",
+    "tsx": "^4.19.2",
+    "ssh-config": "^5.0.3",
```

Comment:
We want the "ssh-config" dep to be a prod dep in the package.json of the package using it (currently `packages/vsce`), rather than a dev dep at the top-level.